content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Accessing HPE Ezmeral Data Fabric Object Storage from Spring Boot S3
+  Micro Service deployed in K3s cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: false
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: https://gravatar.com/avatar/dddba13c1f3712cb7d1ae9a1111d86d6?s=96
+tags:
+  - K3S
+  - Spring Boot
+  - HPE Ezmeral Data Fabric Object Storage
+  - hpe-ezmeral-data-fabric
+  - MapR
+  - Quarkus
+  - Kubernetes
+  - S3
+  - Object Storage
+  - IOT
+---
+Containers and Microservices are transforming Edge and IoT platform use cases which can be deployed in Kubernetes clusters on edge nodes with small footprint and fast processing by persisting data at central location. This data pipeline can be easily accessed by downstream complex analytics applications for further processing. 
```

Comment:
You definitely have the right idea! Awesome! Just some grammatical edits here.  Check out the markup in this snip:
![image](https://user-images.githubusercontent.com/77496261/134178642-578a3785-1e8d-4e64-a72a-1934e1088938.png)

In other words, the "m" in Microservices should be lower case, "which" should be "that", "with" should be "within a", "and" should be "for", and the article "a" should be added just prior to "central".

Reply:
done.